vllm/entrypoints/openai/protocol.py

```diff
@@ -763,7 +767,7 @@ class CompletionRequest(OpenAIBaseModel):
     min_p: Optional[float] = None
     repetition_penalty: Optional[float] = None
     length_penalty: float = 1.0
-    stop_token_ids: Optional[list[int]] = Field(default_factory=list)
+    stop_token_ids: Optional[list[int]] = []
```

Comment:
We need to use default_factory here given that default list is mutable

Reply:
Pydantic base models handle mutable defaults correctly by deep copying them https://docs.pydantic.dev/latest/concepts/fields/#mutable-default-values

Reply:
TIL 😄